config/locales/pa-ur.yml

```
@@ -0,0 +1,441 @@
+    attributes:
+      attachment:
+        hoc_paper_number:
+        isbn:
+        unnumbered_hoc_paper:
+      attachment_data:
+        file:
+      corporate_information_page/corporate_information_page_attachments/attachment:
+        command_paper_number:
+        hoc_paper_number:
+        isbn:
+        title:
+        unique_reference:
+        unnumbered_hoc_paper:
+      corporate_information_page/corporate_information_page_attachments/attachment/attachment_data:
+        file:
+      edition:
+        nation_inapplicabilities:
+      policy_group/policy_group_attachments/attachment/attachment_data:
+        file:
+      response/consultation_response_attachments/attachment:
+        command_paper_number:
+        hoc_paper_number:
+        isbn:
+        title:
+        unique_reference:
+        unnumbered_hoc_paper:
+      response/consultation_response_attachments/attachment/attachment_data:
+        file:
+    nested_attachment_data_field_names:
+      file:
+    nested_attachment_field_names:
+      command_paper_number:
+      hoc_paper_number:
+      isbn:
+      title:
+      unique_reference:
+      unnumbered_hoc_paper:
+  announcements:
+    heading:
+    view_all:
+  attachment:
+    accessibility:
+      full_help_html:
+      intro:
+      request_a_different_format:
+    headings:
+      order_a_copy:
+      order_a_copy_full:
+      published:
+      reference:
+      unnumbered_command_paper:
+      unnumbered_hoc_paper:
+    opendocument:
+      help_html:
+    see_more:
+  broken_links:
+    broken:
+      subheading:
+    caution:
+      subheading:
+    title:
+  change_notes:
+    page_history:
+    published_at:
+    see_all_updates:
+    updated_at:
+  contact:
+    access_and_opening_times:
+    contact_form:
+    email:
+  corporate_information_page:
+    type:
+      link_text:
+        welsh_language_scheme:
+      title:
+        about:
+        about_our_services:
+        access_and_opening:
+        accessible_documents_policy:
+        complaints_procedure:
+        equality_and_diversity:
+        media_enquiries:
+        membership:
+        our_energy_use:
+        our_governance:
+        personal_information_charter:
+        petitions_and_campaigns:
+        procurement:
+        publication_scheme:
+        recruitment:
+        research:
+        social_media_use:
+        staff_update:
+        statistics:
+        terms_of_reference:
+        welsh_language_scheme:
+  date:
+    formats:
+      default:
+      long_ordinal:
+  document_filters:
+    description:
+    no_results:
+      description:
+      title:
+      tna_heading:
+      tna_link:
+    world_locations:
+      all:
+      label:
+  email_signups:
+    new:
+      description:
+      title:
+  feeds:
+    email:
+    feed:
+    get_email_alerts:
+    get_updates_to_this_list:
+    latest_activity:
+    subscribe_to_feed:
+    view_feed:
+  i18n:
+    direction: rtl
+  language_names:
+    pa-ur: Punjabi Shahmukhi
```

```suggestion
    pa-ur: پن٘جابی
```

We need the translation here. Source: https://en.wikipedia.org/wiki/Punjabi_language